valet-key/Readme.md

```diff
@@ -4,47 +4,61 @@ This document describes the Valet Key Pattern example from the guide [Cloud Desi
 
 ## System Requirements
 
-* Microsoft .NET Framework version 4.5
-* Microsoft Visual Studio 2015 Community, Enterprise, or Professional
-* Windows Azure SDK for .NET version 2.9
+* Microsoft .NET Framework 5
+* Microsoft Visual Studio 2019 or Later
+* Windows Azure SDK for .NET release 2021
 
 ## Before you start
 
 Ensure that you have installed all of the software prerequisites.
-
-The example demonstrates operational aspects of applications running in Windows Azure. Therefore, you will need to use the diagnostics tools in order to understand how the code sample works. You **must** ensure that the web and worker roles in the solution are configured to use the diagnostics mechanism. If not, you will not see the trace information generated by the example.
+If you don't have one, provision an Azure Storage account from the Azure Portal.
+Once you storage account is created, add a container to it, name it "valetkeysample".
 
 ## About the Example
  
 This example shows how a client application can obtain a shared access signature with the necessary permissions to write directly to blob storage. For simplicity, this sample focuses on the mechanism to obtain and consume a valet key and does not show how to implement authentication or secure communications.
 
-
 ## Running the Example
 
-You can run this example locally in the Visual Studio Windows Azure emulator. You can also run this example by deploying it to a Windows Azure Cloud Service.
+You can run this example locally in the Visual Studio Windows Azure emulator. You can also run this example by deploying it to a Windows Azure App Service.
 
-* Start Visual Studio using an account that has Administrator privileges ("Run as Administrator").
-* Open the solution you want to explore from the subfolders where you downloaded the examples.
-* Right-click on each role in Solution Explorer, select Properties, and ensure that the role is configured to generate diagnostic information.
+If you want to run this example locally
 
+* Start Visual Studio, set the ValetKet.Web project as startup
+* Edit the appsettings.json and provide values for these settings:
```

Comment:
Please use an ordered list for sequential steps (actions) and an unordered list for concepts/presentation/lists

Reply:
Done